zipkin-lens/src/components/TracePage/TracePage.jsx

```diff
@@ -11,98 +11,50 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
+
+/* eslint-disable no-shadow */
```

Comment:
curious. what is being shadowed.. comment worthy? or is this temporary...

Reply:
`loadTrace` is shadowed..
But this shadow can be avoided.. I'll do so.